devpack/src/main/java/io/neow3j/devpack/annotations/SupportedStandard.java

```diff
@@ -0,0 +1,29 @@
+package io.neow3j.devpack.annotations;
+
+import io.neow3j.devpack.annotations.SupportedStandard.SupportedStandards;
+import io.neow3j.devpack.constants.NeoStandard;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Target;
+
+/**
+ * Used to define features of a smart contract. The attributes will be written to the
+ * contract's manifest file.
```

Comment:
Clarify in the javadoc that only either the `neoStandard` or the `customStandard` attribute should be set but not both.